ai/ai21labs/v0/config/tasks.json

```diff
@@ -0,0 +1,1056 @@
+      "instillUIOrder": 1,
+      "properties": {
+        "input-tokens": {
+          "description": "The input tokens read by AI21labs model",
+          "instillFormat": "number",
+          "instillUIOrder": 2,
+          "title": "Input Tokens",
+          "type": "number"
+        },
+        "output-tokens": {
+          "description": "The output tokens generated by AI21labs model",
+          "instillFormat": "number",
+          "instillUIOrder": 3,
+          "title": "Output Tokens",
+          "type": "number"
+        }
+      },
+      "required": [
+        "input-tokens",
+        "output-tokens"
+      ],
+      "title": "Usage",
+      "type": "object"
+    }
+  },
+  "TASK_TEXT_GENERATION_CHAT": {
+    "instillShortDescription": "Provide text outputs in response to text inputs.",
+    "description": "AI21labs text generation models (often called generative pre-trained transformers or large language models) have been trained to understand natural language, code, and images. The models provide text outputs in response to their inputs. The inputs to these models are also referred to as \"prompts\". Designing a prompt is essentially how you \u201cprogram\u201d a large language model model, usually by providing instructions or some examples of how to successfully complete a task.",
+    "input": {
+      "description": "Input",
+      "instillEditOnNodeFields": [
+        "prompt",
+        "model-name"
+      ],
+      "instillUIOrder": 0,
+      "properties": {
+        "chat-history": {
+          "description": "Incorporate external chat history, specifically previous messages within the conversation. Please note that System Message will be ignored and will not have any effect when this field is populated. Each message should adhere to the format: : {\"role\": \"The message role, i.e. 'system', 'user' or 'assistant'\", \"content\": \"message content\"}.",
+          "instillAcceptFormats": [
+            "structured/chat-messages"
+          ],
+          "instillShortDescription": "Incorporate external chat history, specifically previous messages within the conversation. (Note: Anthropic doesn't support sending images via image-url, use the 'prompt-images' field instead)",
+          "instillUIOrder": 4,
+          "instillUpstreamTypes": [
+            "reference"
+          ],
+          "items": {
+            "$ref": "#/$defs/chat-message"
+          },
+          "title": "Chat history",
+          "type": "array"
+        },
+        "max-new-tokens": {
+          "default": 50,
+          "description": "The maximum number of tokens for model to generate",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUIOrder": 6,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Max new tokens",
+          "type": "integer"
+        },
+        "model-name": {
+          "enum": [
+            "jamba-instruct"
```

Comment:
I think we can provide more cost-efficient option here.

![image](https://github.com/user-attachments/assets/b6832b57-da3e-4f1b-987a-55f175db2d2e)

Reply:
Jurassic series of model will soon be deprecated. source:[link](https://docs.ai21.com/reference/j2-complete-api-ref)

> If you are a new user, we recommend using Jamba Instruct, our newest foundation model. Jurassic (J2) models will be deprecated in the second half of 2024.